airbyte-integrations/connectors/source-s3/scripts/cloud_benchmark.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
```

Comment:
Let's either leave this out of the PR, or just make the slashcommand with this as we discussed with @johnny-schmidt. I'm happy to help you speedrun / pair on it!

Reply:
I may be convinved to just leave this in as the first step and then just not forget to migrate from this in favor of the slash command as a follow-up